### DISCUSSION TITLE
🎬 Rugby Review (ru)

### DISCUSSION THREAD

**Discussion by denandreychuk** [swiftyfinch/Rugby#146] [Demo]
Решил поделиться видео с обзором Rugby. Возможно это поможет кому-то лучше понять как работает CLI, поэтому просто оставлю его здесь :)  Так же, если у @swiftyfinch есть желание, то ссылочку можно добавить куда то в readme, я буду рад!

<a href="https://youtu.be/ohIFW0b6s84" title="Music Caster Video Demo">
  <p align="left">
    <img width="50%" src="https://img.youtube.com/vi/ohIFW0b6s84/maxresdefault.jpg" alt="Основы Cocoapods 05: Как добавить кеширование для подов?"/>
  </p>
</a>

**Comment by swiftyfinch** [comment]
Привет! Спасибо большое за обзор. Я добавил его к остальным видео внизу README.

> Hi! Thank you very much for the review. I added it to the rest of the videos at the bottom of the README.
